### ISSUE TITLE
Enable other extensions to activate ff2mpv

### ISSUE THREAD

**Issue by arthsmn** [woodruffw/ff2mpv#112]
**Is your feature request related to a problem? Please describe.**
I would like to integrate ff2mpv with vimium c, [as requested here](https://github.com/gdh1995/vimium-c/issues/1077). But it seems that ff2mpv doesn't have the interface to allow other extensions to activate it.

**Describe the solution you'd like**
A way that extensions could activate ff2mpv.

**Comment by woodruffw** [comment]
I'm not necessarily opposed to this, but I don't know how to do it. If someone were to send a patch and an explanation of how it works, I'd be happy to merge it and include it in the next release.

**Comment by DanSM-5** [comment]
I just recently added some changes in the code for migrating to mv3 that could be used for this. I'm talking specifically for the listener for `runtime.onMessage` https://github.com/eNV25/ff2mpv/pull/1/files#diff-d59a73ad52dc1dc2b2d687747f1225e38b14324cbd2fa3dd73b2be2b35e42d35R96.

According to the [documentation](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/sendMessage) `runtime.sendMessage` can pass a extensionId to call another extension (ff2mpv in this case). It would be a matter to support a new type of message e.g. `openVideo` with the url as the payload of that message from the extension that wants to call mpv.

```javascript
// External extension
chrome.runtime.sendMessage(ff2mpvExtensionId, { type: 'openVideo', url }, callback);

// ff2mpv
runtime.onMessage.addListener((request, _sender, sendResponse) => {
  const { type, profile, url } = request;
  ...
  
  case 'openVideo':
    ff2mpv(url);
    return sendResponse('ok');
  ...
});
```
I believe that the listener could be included before merging the mv3 migration (so this works with the current version).

**Comment by DanSM-5** [comment]
I've opened a PR with the changes for this to work https://github.com/woodruffw/ff2mpv/pull/113

**Comment by woodruffw** [comment]
Thank you!

**Comment by woodruffw** [comment]
Addressed by #113. Please file a new issue after that's released if you're still seeing problems here.
